app/server/src/routes/routes.ts

```diff
@@ -1,16 +1,94 @@
 import axios from 'axios';
+import passport from 'passport';
 import config from '../resources/config.json';
 
 export const router = (app => {
-    app.get('/', (request, response) => {
-        response.json({
-            message: 'Welcome to beebop!'
+    app.get('/',
+        (request, response) => {
+            response.json({
+                message: 'Welcome to beebop!'
+            });
+        }
+    );
+
+    app.get('/version',
+        getVersionInfo);
+
+    app.get('/login/google',
+        passport.authenticate('google', { scope: ['profile'] }));
+
+    app.get('/login/github',
+        passport.authenticate('github', { scope: ['profile'] }));
+
+    app.get('/login/mock', passport.authenticate('mock'), (req, res) => {
```

Comment:
Again, if this is just here for testing, it would be better to move this into testing code, if possible, and it's not really testing the other routes. I think mock strategy is really a way of setting up a mocked backend to test that front end code is interacting with it as expected, not really a way of testing the server itself.